### PR TITLE
WINCE: Fix compilation

### PR DESCRIPTION
**Pull request by ccawley2011**
This just about compiles with [this Docker image](https://github.com/ccawley2011/scummvm-docker/tree/wince/workers/wince), but has not been tested. However, because of the large amount of bitrot resulting from the port's age and lack of a maintainer, I would be surprised if it works at all. As such, it might not be worth keeping the Windows CE port around, especially since as-is, it blocks other developments such as scaler plugins.

### CODE REVIEW

File: backends/platform/wince/CEDevice.h [hunk 1]
```diff
@@ -23,6 +23,8 @@
 #ifndef CEDEVICE_H
 #define CEDEVICE_H
 
+#include <windows.h>
```
Comment (bluegr) [inline backends/platform/wince/CEDevice.h]: You’ve already included this in the .cpp file. Is it also needed here?

Comment (ccawley2011) [reply backends/platform/wince/CEDevice.h]: Fixed.

File: backends/plugins/win32/win32-provider.cpp [hunk 2]
```diff
@@ -36,7 +36,7 @@
 class Win32Plugin : public DynamicPlugin {
 private:
 	static const TCHAR* toUnicode(const char *x) {
-	#ifndef _WIN32_WCE
+	#ifndef UNICODE
```
Comment (bluegr) [inline backends/plugins/win32/win32-provider.cpp]: (Same comment, in a review)
Why has this been changed? The former check for _WIN32_WCE is done in other places in the code

Comment (ccawley2011) [reply backends/plugins/win32/win32-provider.cpp]: This is done to match a similar function in [backends/fs/windows/windows-fs.cpp](https://github.com/scummvm/scummvm/blob/master/backends/fs/windows/windows-fs.cpp#L99).

**Comment by bluegr** [comment]
Many thanks for this, once more! Hopefully, this will help bring the port up to date.

Merging

**Pull request merged**